src/main/webapp/resources/angular-templates/admin/partials/event/detail.html

```diff
@@ -1,5 +1,23 @@
 <div class="fa fa-refresh fa-spin fa-2x text-muted" data-ng-if="!event">
 </div>
+<style>
+    #sold{
```

Comment:
Move style in admin.css please.

Additionally, they should be class selector, not id selector, see below why.